STATUS: INLINE SUCCESSFUL
MODIFIED: com.alibaba.json.bvt.serializer.MaxBufSizeTest.test_max_buf
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/fastjson/src/test/java/com/alibaba/json/bvt/serializer/MaxBufSizeTest.java
CLEANER: com.alibaba.json.bvt.serializer.TestGroup100Case0.test310
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/fastjson/src/test/java/com/alibaba/json/bvt/serializer/TestGroup100Case0.java
POLLUTER: com.alibaba.json.bvt.serializer.SerializeWriterTest_14.test_writer_2
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/fastjson/src/test/java/com/alibaba/json/bvt/serializer/SerializeWriterTest_14.java
ORIGINAL CLEANER SIZE: 29
NEW CLEANER SIZE: 1
ELAPSED TIME: 910.7439999580383
==========================
@@ -12,0 +12,3 @@
+ {
+     com.alibaba.fastjson.serializer.SerializeWriter serializeWriter26 = new com.alibaba.fastjson.serializer.SerializeWriter();
+ }
